### PR TITLE
MTA-3416: Adding requirements for Java analyzer and discovery task

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [MTA-3416](https://issues.redhat.com/browse/MTA-3416)

### PREVIEW

* [Adding minimum requirements for Java analyzer and discovery](https://anarnold97.github.io/MTA-Preview/MTA/MTA-3416.html#mta-requirements-for-java-analyzer_user-interface-guide)

### CODE REVIEW

**Comment by shawn-hurley** [comment]
I would make it clear that this is also the default, that you don't have to set it.

**Review by shawn-hurley — APPROVED** [review]

File: docs/topics/mta-requirements-for-java-analyzer.adoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-requirements-for-java-analyzer_{context}"]
+= Adding minimum requirements for Java analyzer and discovery
+
+There is a minimum required for the Java analyzer, and as a result, the  discovery task.
+
+The minimum requirement is 1.5 GiB. However, this minimum requirement is not unadvisable. The suggested requirement is a minimum of 2 GiB.
```
Comment (dymurray) [inline docs/topics/mta-requirements-for-java-analyzer.adoc:11]: "is not unadvisable"?

I don't think this makes sense. I would just say the recommended and default option is 2GiB, but could be lowered to 1.5GiB though not recommended.

Comment (anarnold97) [inline docs/topics/mta-requirements-for-java-analyzer.adoc:11]:
```suggestion
The suggested requirement is a default minimum of 2 GiB. 

The minimum requirement can be lowered to  1.5 GiB. However, this minimum requirement is not recommended. 
```

File: docs/topics/mta-requirements-for-java-analyzer.adoc [hunk 2]
```diff
@@ -8,7 +8,9 @@
 
 There is a minimum required for the Java analyzer, and as a result, the  discovery task.
 
-The minimum requirement is 1.5 GiB. However, this minimum requirement is not unadvisable. The suggested requirement is a minimum of 2 GiB.
+The suggested requirement is a default minimum of 2 GiB. 
+
+The minimum requirement can be lowered to  1.5 GiB. However, this minimum requirement is not recommended. 
 
 This minimum requirement can be set in `provider_java_container_limits_memory` and `provider_java_container_requests_memory` options in the Tackle custom resource.
```
Comment (shawn-hurley) [inline docs/topics/mta-requirements-for-java-analyzer.adoc:15]: The minimum requirement is already set to 2GB if you would like to increase, you can change the values, or something like that. 

I would suggest also that we say something like "to gurantee scheduling has the correct space, requests and limits should be the same.

**Review by sshveta — COMMENTED** [review]
I will defer to Shawn for review on this .

File: docs/topics/mta-requirements-for-java-analyzer.adoc (hunk 2 above)
Comment (anarnold97) [reply docs/topics/mta-requirements-for-java-analyzer.adoc]: Updated

File: docs/topics/mta-requirements-for-java-analyzer.adoc [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-requirements-for-java-analyzer_{context}"]
+= Adding minimum requirements for Java analyzer and discovery
+
+There is a minimum requirement for the Java analyzer, and also the discovery task, which by default is set to 2 GiB.
+
+While this minimum requirement can be lowered to  1.5 GiB, it is not recommended.
```
Comment (anarnold97) [inline docs/topics/mta-requirements-for-java-analyzer.adoc:11]:
```suggestion
While this minimum requirement can be lowered to 1.5 GiB, it is not recommended.
```

**Pull request merged**